config/earlyapp.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=GP2.0 Early application
 DefaultDependencies=no
-Requires=ias-earlyapp.service cbc_attach.service earlyapp-setup.service earlyapp-setup-ipu.service
-After=ias-earlyapp.service cbc_attach.service earlyapp-setup.service earlyapp-setup-ipu.service
+Requires=ias-earlyapp.service cbc_attach.service earlyapp-setup.service earlyapp-setup-ipu.service earlyapp-setup-audio.service
+After=ias-earlyapp.service cbc_attach.service earlyapp-setup.service earlyapp-setup-ipu.service earlyapp-setup-audio.service
```

Comment:
Please note that currently the application support two different environments.
  1. earlayapp.service # Uses MSDK, ICI, ALSA directly.
  2. earlyapp_gst.service # Uses Gstreamer by default.

Reply:
Yes that was my first thought though, if this is a platform specific, I think you don't have to.